models/notification.schema.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+  isNew: Boolean,
```

Comment:
What is this for?

Reply:
It's the boolean for whether the notification is new or has been closed. I wasn't sure what to name it

Reply:
i think we actually don't need this field, we can just delete notifications if they have been closed.